common/static/common/js/karma.common.conf.js

```diff
@@ -0,0 +1,216 @@
+// Common configuration for Karma
+/* jshint node: true */
+'use strict';
```

Comment:
@andy-armstrong I had fixed all jshint issues. The quality check is green, you can see at the bottom. If I am missing anything can you please elaborate?